lib/element.js

```diff
@@ -242,7 +242,7 @@ ElementArrayFinder.prototype.get = function(index) {
         i = parentWebElements.length + i;
       }
       if (i < 0 || i >= parentWebElements.length) {
-        throw new Error('Index out of bound. Trying to access element at ' +
+            console.warn('Index out of bound. Trying to access element at ' +
```

Comment:
I still think `.get` should fail for out of bounds indexes, instead of returning `undefined`